src/__tests__/element-queries.js

```diff
@@ -2,8 +2,8 @@ import 'jest-dom/extend-expect'
 import {render} from './helpers/test-utils'
 
 beforeEach(() => {
-  window.Cypress = null;
-});
+  window.Cypress = null
```

Comment:
Off-topic but why null, not undefined? @kentcdodds

Reply:
Someone else did that I think. I probably would have set it to undefined, but I didn't care enough to ask them to change it.